sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -894,12 +895,14 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
 
           sql(insertString.toLowerCase(Locale.ROOT))
           sql(insertString.toUpperCase(Locale.ROOT))
+          spark.sessionState.catalog.alterTableStats(TableIdentifier("test1"), None)
```

Comment:
what are we doing here?

Reply:
Reuse this UT to test update table stats code

Reply:
Does it test anything? It just invokes `alterTableStats` but does no verification.

Reply:
Remove this change